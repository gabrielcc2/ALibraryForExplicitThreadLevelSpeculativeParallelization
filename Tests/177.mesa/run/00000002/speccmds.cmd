-u /home/gabriel/Escritorio/SPEC_CPU2000_CD_2/benchspec/CFP2000/177.mesa/run/00000002
../00000002/mesa_base.x86_linux -frames 10 -meshfile mesa.in -ppmfile mesa.ppm
