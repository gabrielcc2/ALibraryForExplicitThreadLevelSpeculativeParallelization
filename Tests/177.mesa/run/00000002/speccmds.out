running commands in speccmds.cmd 1 times
runs started at 1312934523, 211474000, Tue Aug  9 19:32:03 2011
run 1 started at 1312934523, 211565000, Tue Aug  9 19:32:03 2011
child started: 0, 1312934523, 211579000, 10390, '../00000002/mesa_base.x86_linux -frames 10 -meshfile mesa.in -ppmfile mesa.ppm'
child finished: 0, 1312934524, 688017000, 1, 476438000, 10390, 0
run 1 finished at: 1312934524, 688027000, Tue Aug  9 19:32:04 2011
run 1 elapsed time: 1, 476462000, 1.476462000
runs finished at 1312934524, 688076000, Tue Aug  9 19:32:04 2011
runs elapsed time: 1, 476602000, 1.476602000
